maintainers/security-reports.md

```diff
@@ -0,0 +1,25 @@
+# Handling security reports
+
+Reports can be expected to be submitted following the [security policy](https://github.com/NixOS/nix/security/policy), but may reach maintainers on various other channels.
+
+In case a vulnerability is reported:
+
+1. [Create a GitHub security advisory](https://github.com/NixOS/nix/security/advisories/new)
+
+   > [!IMPORTANT]
+   > Add the reporter as a collaborator so they get notified of all activities.
+
+   Note deadlines if applicable.
```

Comment:
Was digging through the OWASP cheatsheet for vulnerability disclosures and thought these additional items might be important for the report. (See [here](https://cheatsheetseries.owasp.org/cheatsheets/Vulnerability_Disclosure_Cheat_Sheet.html#initial-report))

```suggestion
   In addition to the details in the advisory template, the initial report should:
   
   - Include sufficient details of the vulnerability to allow it to be understood and reproduced.
   - Redact any personal data.
   - Set a deadline (if applicable).
   - Provide proof of concept code (if available).
   - Reference any further reading material that may be appropriate.
```

Reply:
Oh yeah, thanks!